<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>478</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

background-color: rgb(0, 78, 236);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>341</width>
      <height>324</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="Ecuacion1">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 12pt &quot;MS Serif&quot;;
    font-weight: bold;
    background-color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
    text-align: center; 
	border: 2px solid #FF0000;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Ecuacion2">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 12pt &quot;MS Serif&quot;;
    font-weight: bold;
    background-color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
    text-align: center; 
	border: 2px solid #FF0000;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Ecuacion3">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 12pt &quot;MS Serif&quot;;
    font-weight: bold;
    background-color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
    text-align: center; 
	border: 2px solid #FF0000;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Ecuacion4">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 12pt &quot;MS Serif&quot;;
    font-weight: bold;
    background-color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
    text-align: center; 
	border: 2px solid #FF0000;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Ecuacion5">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 12pt &quot;MS Serif&quot;;
    font-weight: bold;
    background-color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
    text-align: center; 
	border: 2px solid #FF0000;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Ecuacion6">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 12pt &quot;MS Serif&quot;;
    font-weight: bold;
    background-color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
    text-align: center; 
	border: 2px solid #FF0000;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(255, 255, 255);

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    font: 12pt &quot;MS Serif&quot;;
    font-weight: bold;
  </string>
     </property>
     <property name="text">
      <string>Ingrese la ecuacion:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtEcuacion">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">

 
    color: #000;
    border: 1px solid #000;
    border-radius: 10px;
    font:20px &quot;MS Serif&quot;;
    padding: 5px 20px;
    background-color: #CCC;
</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Serif'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:15px;&quot;&gt;ej: 8-2+7=13&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="oKButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #FFD920;
    color: #333;
    border: 2px solid #000;
    border-radius: 10px;
    font: bold 10px &quot;MS Serif&quot;;
    padding: 5px 20px;
}

QPushButton:hover {
    background-color: #FFEC3A;
    color: #000;
}

QPushButton:pressed {
    background-color: #FFC700;
}</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
